AmazonChimeSDK/AmazonChimeSDK/ingestion/IngestionEventConverter.swift

```diff
@@ -8,62 +8,79 @@
 
 import Foundation
 
+private typealias EAName = EventAttributeName
+
 /// `IngestionEventConverter` converts data from payload into `MeetingEventItem`/`DirtyEventItem`or vice versa.
 @objcMembers public class IngestionEventConverter: NSObject {
-    private let nameKey = "name"
-    private let timestampKey = "ts"
-    private let idKey = "id"
-    private let ttlKey = "ttl"
 
     public override init() {
         super.init()
     }
 
-    func toIngestionMeetingEvent(event: SDKEvent, ingestionConfiguration: IngestionConfiguration) -> IngestionMeetingEvent {
+    func toIngestionMeetingEvent(event: SDKEvent,
+                                 ingestionConfiguration: IngestionConfiguration) -> IngestionMeetingEvent {
         let eventAttributes = event.eventAttributes
         let meetingConfig = ingestionConfiguration.clientConfiguration as? MeetingEventClientConfiguration
 
         var meetingStatus: String?
         if let status = eventAttributes[EventAttributeName.meetingStatus] as? MeetingSessionStatusCode {
             meetingStatus = String(describing: status)
+        } else if let status = eventAttributes[EventAttributeName.meetingStatus] as? String {
+            // MeetingStatus is emitting as string in DefaultAudioClientObserver.notifyFailed(status: MeetingSessionStatusCode)
```

Comment:
Hmmm I don't see it emitting as string [here](https://github.com/aws/amazon-chime-sdk-ios/blob/b20fe3bdafa482650d9901395ee94a4a2ba484d1/AmazonChimeSDK/AmazonChimeSDK/internal/audio/DefaultAudioClientObserver.swift)

Reply:
You are right, I looked it wrong. Will update.